modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp

```diff
@@ -10,6 +10,8 @@
 <%
 EditKBArticleDisplayContext editKBArticleDisplayContext = new EditKBArticleDisplayContext(kbGroupServiceConfiguration, liferayPortletRequest, liferayPortletResponse, portletConfig);
 
+KBArticle kbArticle = editKBArticleDisplayContext.getKBArticle();
```

Comment:
Don't get the kbArticle because it can be null when creating a new article. Better create methods for each kbArticle you need (following the DisplayContext structure).